src/components/Request/Anthonyview.jsx

```diff
@@ -0,0 +1,137 @@
+import react from 'react';
+import React, {Component} from 'react';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+import moment from 'moment';
+
+export default class Anthonyview extends react.Component{
+    constructor(props){
+        super(props);
+        this.state = {
+          topic: '',
+          description: '',
+          time: '',
+          slots: []
+        }
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+        this.handleTimeSlot = this.handleTimeSlot.bind(this);
+    
+      }
+      handleChange(e) {
+        this.setState({
+          [e.target.name] : e.target.value
+        })
+      }
+    
+      handleTimeSlot(e) {
+    
+        const index = e.target.selectedIndex - 1;
+          this.setState({
+            time: e.target.value,
+            slotId: this.state.slots[index].id,
+            selectedSlot: this.state.slots[index]
+          })
+      }
+    
+      handleSubmit(e) {
+        axios.post(`/api/Visitors/${this.props.user.id}/aptRequests`, {
+          "topicSummary": this.state.topic,
+          "studentName": this.props.user.firstName + ' ' + this.props.user.lastName,
+          "email": this.props.user.email,
+          "issueDescription": this.state.description,
+          "time": this.state.time,
+          "slotId": this.state.slotId,
+          "instructorId": "anthony"
+        }).then((response) => {
+          this.setState({
+            alert: 'Awesome possum! Your appointment has been requested.',
+            error: false,
+            topic: '',
+            description: '',
+            time: ''
+          })
+        }).catch ((err) => {
+          this.setState({
+            alert: 'Sorry, something went wrong! Try again',
+            error: true
+          })
+        })
+        let selecturd = this.state.selectedSlot;
+        let slotties = this.state.slots;
+
+        for(var i =0; i < slotties.length; i++){
+            if(slotties[i].id === selecturd.id){
+              slotties.splice(i, 1);
+            }
+        } 
+
+        this.setState({
+          slots:slotties
+        });
+
+        const instructorEmail = `${this.state.selectedSlot.instructorId}@origincodeacademy.com`
+        const time = this.state.time;
+        const studentName = (this.props.user.firstName + ' ' + this.props.user.lastName);
+        axios.post(`/api/AptRequests/emailAdmin`, {
```

Comment:
This email should go out after the previous `axios.post`. What happens if the POST request fails? We'll still get an email